docs/reference/migration/migrate_7_13.asciidoc

```diff
@@ -117,13 +117,62 @@ aggregations are rarely useful and often unintended.
 [%collapsible]
 ====
 *Details* +
-Support for multiple paths in the `path.data` setting is now deprecated. We
-introduced this option as a way to support multi-disk setups. It has since been
-a source of user complaints due to confusing and unintuitive behavior.
+The `path.data` setting accepts a list of data paths, but if you specify
+multiple paths then the behaviour is unintuitive and usually does not give the
+desired outcomes. Support for multiple data paths is now deprecated and will be
+removed in 8.0.0.
 
 *Impact* +
-Specify a single path in `path.data`. To use multiple disks, use a RAID
-hardware configuration or similar hardware solution.
+Specify a single path in `path.data`. If needed, you can create a filesystem
+which spans multiple disks with a hardware virtualisation layer such as RAID,
+or a software virtualisation layer such as Logical Volume Manager (LVM) on
+Linux or Storage Spaces on Windows. If you wish to use multiple data paths on a
+single machine then you should run one node for each data path.
+
+If you currently use multiple data paths in a
+<<high-availability-cluster-design,highly available cluster>> then you can
+migrate to a setup that uses a single path for each node without downtime using
+a process similar to a <<restart-cluster-rolling,rolling restart>>: shut each
+node down in turn and replace it with one or more nodes each configured to use
+a single data path. At each step this process will discard the data spread
+across the data paths on the node that is being replaced, so you must make sure
+that your cluster health is never `red` during this process and we recommend
```

Comment:
I think they should aim for green health between doing any node?

Also, I wonder if we can add some minimal guidance on planning the size of the cluster to be able to contain all their data. Would be sad to have them half-way through this and then discover that there is not enough room. Not sure if it is too much "stating the obvious"?

Reply:
Agreed. It got complicated enough to deserve a stepwise recipe so that's what I did, see 769eba3d9b2.